IdentityCore/src/cache/token/MSIDCredentialCacheItem.m

```diff
@@ -187,7 +187,7 @@ - (NSDictionary *)jsonDictionary
     dictionary[MSID_HOME_ACCOUNT_ID_CACHE_KEY] = _homeAccountId;
     dictionary[MSID_CLIENT_INFO_CACHE_KEY] = _clientInfo.rawClientInfo;
     dictionary[MSID_SPE_INFO_CACHE_KEY] = _additionalInfo[MSID_SPE_INFO_CACHE_KEY];
-    dictionary[MSID_EXTENDED_EXPIRES_ON_CACHE_KEY] = [_additionalInfo[MSID_EXTENDED_EXPIRES_ON_CACHE_KEY] msidDateToTimestamp];
+    dictionary[MSID_EXTENDED_EXPIRES_ON_LEGACY_CACHE_KEY] = [_additionalInfo[MSID_EXTENDED_EXPIRES_ON_LEGACY_CACHE_KEY] msidDateToTimestamp];
```

Comment:
nit: can we remove legacy from the key name since it's used everywhere?

Reply:
done